src/platform_impl/macos/window_delegate.rs

```diff
@@ -305,6 +305,28 @@ declare_class!(
             options
         }
 
+        /// Invoked when miniaturize
+        #[sel(windowDidMiniaturize:)]
+        fn window_did_miniaturize(&mut self, _: Option<&Object>) {
+            trace_scope!("windowDidMiniaturize:");
+
+            self.queue_event(WindowEvent::Configured {
+                size: Default::default(),
```

Comment:
What do you want here as size, @kchibisov?

Reply:
You could send the current size.